.travis.yml

```diff
@@ -0,0 +1,53 @@
+language: android
+jdk:
+  - oraclejdk8 
+
+android:
+  components:
+    # Uncomment the lines below if you want to
+    # use the latest revision of Android SDK Tools
+    - tools
+    - platform-tools
+    - tools
+
+    # The BuildTools version used by your project
+    - build-tools-24.0.2
+    - build-tools-26.0.2
+
+    # The SDK version used to compile your project
+    - android-22
+    - android-26
+
+    # Additional components
+    - extra-google-google_play_services
+    - extra-google-m2repository
+    - extra-android-m2repository
+    - addon-google_apis-google-26
+
+    # Specify at least one system image,
+    # if you need to run emulator(s) during your tests
+    - sys-img-armeabi-v7a-android-22
+    # - sys-img-armeabi-v7a-android-17
+
+before_script:
+  # Create and start emulator
+  - android list targets
+  - echo no | android create avd --force -n test -t android-22 --abi armeabi-v7a
+  - emulator -avd test -no-skin -no-audio -no-window &
+  #- adb wait-for-device
+  - android-wait-for-emulator
+  - adb shell input keyevent 82 &
+  
+before_install:
+  - cd AppHome
+  - chmod +x gradlew
+  - echo yes | $ANDROID_HOME/tools/bin/sdkmanager "build-tools;27.0.3"
+  
+notifications:
+  email: false
+  slack:
+    on_success: change
+    rooms:
+      - deltadak:SDesvinRRI6WqYut1gxC1Qbs#apphome
```

Comment:
Are you sure you encrypted this? Otherwise in theory everyone could send notifications to our Slack :)

Reply:
Right. Eh. That was the next step. Honestly. I guess I was just happy that it worked...